infra/kdp-core/resources/kdp-cloudtty.cue

```diff
@@ -0,0 +1,447 @@
+package main
+
+_CloudTtyName: "cloudtty"
+
+_kdpCloudTty: {
+	name: parameter.namePrefix + _CloudTtyName
+	type: "helm"
+	properties: {
+		url:             "\(parameter.helmURL)"
+		chart:           _CloudTtyName
+		releaseName:     parameter.namePrefix + _CloudTtyName
+		repoType:        "oci"
+		version:         "0.5.7"
```

Comment:
L13 and L77 use the same value, we should define a variable for them.

Reply:
chart values and image values are not the same, so the same variable cannot be used here